app/renderer/components/tooltip.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+module.exports = {
+    props: ['color'],
+    template: `
+    <div class="tooltip">
+        <div class="tooltip-text">
+            {{ status }}
+        </div>
+    </div>
+    `,
+    computed: {
+        status() {
+            switch (this.color) {
```

Comment:
We shouldn't be using a prop called "color" for something that is not defining a color.

For this, we should be using the enumeration defined app/common/task_status
For example:
```js
case TaskStatus.idle:
     return "Idle" 
```

Instead of the prop "color" we should send "status" to the tooltip component